### DISCUSSION TITLE
Right way to apply shadow to button component

### DISCUSSION THREAD

**Discussion by nouman91** [mantinedev/mantine#664] [Q&A] [answered]
I have two questions for applying shadows to button components 

Q1) Is this the right way to apply shadow to the button component?

` <Button
          sx={(theme) => ({
            "box-shadow": theme.shadows.lg,
          })}
          size="lg"
        >
          Settings{" "}
        </Button>`
        
     
Q2) Can we have a default **shadow** prop along with other default props?

**Comment by rtivital** [comment, marked as answer]
1) yes
2) We will consider adding it to the theme
